internal/api/api.go

```diff
@@ -195,3 +199,19 @@ func startMonitoringServer(logger zerolog.Logger) {
 
 	logger.Info().Str("port", "8888").Msg("Started monitoring web server.")
 }
+
+func httpMetricsMiddleware(c *fiber.Ctx) error {
```

Comment:
only thing to be aware of is will this catch the path with the parameter or just the path